### PR TITLE
Upload picoMini writeups

### PR DESCRIPTION
**Pull request by annaahsu**

### CODE REVIEW

**Comment by DarinMao** [comment]
Hi, thanks for contributing

This looks pretty good, but as others may want to add more writeups for this CTF in the future, could you change your title and slug to be more specific to yourself?

Thanks!

File: writeups/picomini-2021/picomini-2021.md [hunk 1]
```diff
@@ -0,0 +1,76 @@
+---
+title: picoMini by redpwn 2021
```
Comment (DarinMao) [inline writeups/picomini-2021/picomini-2021.md]: consider including challenge names in title

File: writeups/picomini-2021/picomini-2021.md [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+date: 2021-05-15
```
Comment (DarinMao) [inline writeups/picomini-2021/picomini-2021.md]: from the future? 👀

File: writeups/picomini-2021/picomini-2021.md [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+slug: /writeups/picomini-redpwn
+excerpt: Writeups for login (web) and advanced-potion-making (forensics).
+author: Anna Hsu
+---
+The picoCTF team decided to host picoMini by redpwn in the midst of AP and finals cram season, which was unfortunate. However, I did solve their two lowest level challenges, and their detailed step-by-step solutions are below. If you're a beginner, I hope you find this helpful!
```
Comment (DarinMao) [inline writeups/picomini-2021/picomini-2021.md]: no, we decided this ;)

File: writeups/picomini-2021/picomini-2021.md [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+
+## web - login
```
Comment (DarinMao) [inline writeups/picomini-2021/picomini-2021.md]: top-level heading should be 1 # only

**Comment by joshdabosh** [comment]
lgtm

**Review by DarinMao — DISMISSED** [review]
Some more things to consider: 
- perhaps change the folder name 
- maybe split into two separate writeups, though they are quite short so this is not necessary

**Comment by joshdabosh** [comment]
still lgtm :)

**Review by DarinMao — APPROVED** [review]
lgtm

**Pull request merged**